### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by bl-ue**

### CODE REVIEW

**Review by d2dyno1 — DISMISSED** [review]

File: Files/Views/SettingsPages/Experimental.xaml [hunk 1]
```diff
@@ -55,20 +55,20 @@
...
+                    Value="{Binding PreemptiveCacheParallelLimit, Mode=TwoWay}"
+                    Minimum="1"
                     HeaderTemplate="{StaticResource CustomHeaderStyle}"
                     IsEnabled="{x:Bind converters:MultiBooleanConverter.AndConvert(viewModel.UseFileListCache, viewModel.UsePreemptiveCache), Mode=OneWay}"/>
-                
+
                 <ToggleSwitch
                     x:Uid="SettingsShowMultiselectOption"
-                    Header="Enable multiselect options (currently only abailable in the tiles and grid view layout modes)"
+                    Header="Enable multiselect options (currently only available in the tiles and grid view layout modes)"
```
Comment (yaichenbaum) [inline Files/Views/SettingsPages/Experimental.xaml:71]: Can you update this string in the `en-US` resource file as well?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/Views/SettingsPages/Experimental.xaml (hunk 1 above)
Comment (bl-ue) [reply Files/Views/SettingsPages/Experimental.xaml:71]: @yair100 as a matter of fact, this is the only occurrence of "abailable"&mdash;all other strings are correct.

Comment (bl-ue) [reply Files/Views/SettingsPages/Experimental.xaml:71]: (I did a global find+replace across all files so all occurrence were caught)

File: Files/Extensions/TaskExtensions.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ internal static class TaskExtensions
 #pragma warning disable RCS1175 // Unused this parameter.
 #pragma warning disable IDE0060 // Remove unused parameter
         /// <summary>
-        /// This function is to explicitly state that we know that we're runnign task without awaiting.
+        /// This function is to explicitly state that we know that we're running task without awaiting.
         /// This makes visual studio to drop warning, but the programmer intent is still clearly stated.
```
Comment (bl-ue) [inline Files/Extensions/TaskExtensions.cs:11]:
```suggestion
        /// This makes Visual Studio to drop warning, but the programmer intent is still clearly stated.
```

Comment (bl-ue) [inline Files/Extensions/TaskExtensions.cs:11]:
```suggestion
        /// This makes Visual Studio to drop warning, but the programmer intent is still clearly stated.
```

**Review by yaichenbaum — APPROVED** [review]
@bl-ue Great work! LGTM

**Pull request merged**